doc/source/myrocks/zenfs.rst

```diff
@@ -138,53 +152,54 @@ Configuration
 
       For more information, see :ref:`enable-apparmor`.
 
-#. Initialize ZenFS on ``nvme0n2``.
-
-   .. code-block:: bash
-	
-	  $ sudo -H -u mysql zenfs mkfs --zbd=nvme0n2 --aux_path=/var/lib/mysql_zenfs_aux_ nvme0n2 --finish_threshold=0 --force
-		
    .. note::
 	
 		 If you must configure ZenFS to use a directory inside ``/var/lib`` (owned by ``root:root`` without write permissions for other user accounts), edit your AppArmor profile (described in an earlier step), if needed, and do the following steps manually:
 		
-		 #. Create the ``aux_path`` for ``nvme0n2``:
+		 #. Create the ``aux_path`` for ``<block_device_name>``:
 		
 		    .. code-block:: bash
 		
-			    $ sudo mkdir /var/lib/mysql_zenfs_aux_ nvme0n2
+			    $ sudo mkdir /var/lib/mysql_aux_<short_block_device_name>
 			
 		 #. Change the ownership of the ``aux_path``:
 		
 		    .. code-block:: bash
 		
-			    $ sudo chown mysql:mysql /var/lib/mysql_zenfs_ nvme0n2
+			    $ sudo chown mysql:mysql /var/lib/mysql_aux_<short_block_device_name>
 			
-		 #. Set the permissions for the ``aux_path`` for ``nvme0n2``:
+		 #. Set the permissions for the ``aux_path`` for ``<block_device_name>``:
 		
 		    .. code-block:: bash
 		
-			    $ sudo chmod 750 /var/lib/mysql_zenfs_aux_ nvme0n2
+			    $ sudo chmod 750 /var/lib/mysql_aux_<short_block_device_name>
 
- 		 #. Create the file system:
-		
-		    .. code-block:: bash
-		
-			    $ sudo -H -u mysql zenfs mkfs     
+
+#. Change the ownership of ``<block_device_name>``.
+
+   .. code-block:: bash
+
+          $ sudo chown mysql:mysql <block_device_name>
+
+#. Initialize ZenFS on ``<block_device_name>``.
+
+   .. code-block:: bash
+
+	  $ sudo -H -u mysql zenfs mkfs --zbd=<short_block_device_name> --aux_path=/var/lib/mysql_aux_<short_block_device_name> --finish_threshold=0 --force
 		
 #. Stop *Percona Server for MySQL*:
 
    .. code-block:: bash
 	
 	  sudo service mysql stop 
 		
-#. Edit my.cnf. Add the following line to the "[mysqld]" section: 
+#. Edit /etc/my.cnf or rather the mysqld.cnf where /etc/my.cnf is pointing to (e.g. /etc/mysql/mysql.conf.d/mysqld.cnf). Add the following line to the "[mysqld]" section:
```

Comment:
Typically, we do ask the user to edit the my.cnf option file. Yes, the files are linked but we should only use my.cnf.